bin/post_log_message.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+import argparse
+import json
+from os import getenv
+import requests
+import sys
+
+
+def exit_verbosely(message, code=2):
+    """Call with message to exit with details echoed to stderr"""
+    print(message, file=sys.stderr)
```

Comment:
We should probably standardize on using the logger, once we figure out an easy way to configure it

Reply:
Well, that's the rub.  It's a simple script, and logging ain't.  STDERR should be easy to capture, no?